src/samples.jl

```diff
@@ -523,7 +523,7 @@ function Base.show(io::IO, samples::Samples)
         duration_in_seconds = size(samples.data, 2) / samples.info.sample_rate
         duration_in_nanoseconds = round(Int, duration_in_seconds * 1_000_000_000)
         println(io, "Samples (", TimeSpans.format_duration(duration_in_nanoseconds), "):")
-        println(io, "  info.kind: ", repr(samples.info.kind))
+        println(io, "  info.sensor_type: ", repr(samples.info.sensor_type))
```

Comment:
should we add one for `sensor_label`?
```suggestion
        println(io, "  info.sensor_type: ", repr(samples.info.sensor_type))
        println(io, "  info.senor_label: ", repr(samples.info.senor_label))
```

Reply:
sensor_label purposefully isn't a field of Samples